core/sawtooth/manage/daemon.py

```diff
@@ -77,13 +69,11 @@ def _construct_start_command(self, node_args):
         if not os.path.exists(config_dir):
             os.makedirs(config_dir)
```

Comment:
"config_dir" doesn't seem to even be used elsewhere in the code now.  Should the creation of this directory, and references to it, just be removed altogether?